doc/source/examples/incompressible-flow/2d-transient-flow-around-cylinder/2d-transient-flow-around-cylinder.rst

```diff
@@ -0,0 +1,266 @@
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+          set kinematic viscosity            = 0.005
+        end
+    end
+
+Forces
+~~~~~~
+
+Since we want to study the time evolution of the drag and lift coefficients, the force acting on the boundaries must be computed. We thus use the ``forces`` subsection:
+
+.. code-block:: text
+
+   subsection forces
+      set verbosity             = verbose
+      set calculate force       = true
+      set calculate torque      = false
+      set force name            = force
+      set output precision      = 10
+      set calculation frequency = 1
+      set output frequency      = 1
+   end
+
+As we set ``calculation frequency`` to 1, the forces on each boundary are computed at each time step and written in the file specified by the field ``force name``.
+
+.. note::
+
+  The drag and lift coefficients are obtained with the forces acting on the wall of the cylinder (i.e., ``f_x`` and ``f_y``  written in the file ``forces.00.dat``) :
+
+  .. math::
+
+    C_D = \frac{2 f_x}{\rho U_\infty^2 D} \text{ and } C_L = \frac{2 f_y}{\rho U_\infty^2 D}
+
+  where :math:`\rho = 1`. This way, we can obtain the evolution in time of both coefficients.
+
+.. warning::
+
+  The computational cost of writing this output file at each time step by setting ``output frequency`` to 1 can be significant, as explained in `Force and torque calculation <https://lethe-cfd.github.io/lethe/parameters/cfd/force_and_torque.html>`_. However, in the present example, the computational cost is still fairly low so ``output frequency = 1`` is used.
+
+
+Running the simulation
+----------------------
+The simulation is launched in parallel using 10 CPUs, as explained in `2D Transient flow around an Ahmed body <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-transient-around-ahmed-body/2d-transient-around-ahmed-body.html>`_ :
+
+.. code-block:: text
+
+  mpirun -np 10 gls_navier_stokes_2d cylinder.prm
+
+.. warning::
+
+  The estimated time to simulate 200 seconds is about 2 hours 5 minutes with 10 CPUs.
+
+Results
+-------
+
+The time evolution of the drag and lift coefficients is obtained from a Gnuplot script available in the example folder:
+
+.. image:: images/CL-CD.png
+    :alt: CD and CL evolution in time
+    :align: center
+    :name: CD-CL
+
+
+Using the FFT of the CL for the last 100 seconds, we can obtain the frequency :math:`f_v` at which the vortices are shed :
```

Comment:
Does FFT stand for fast-Fourier transform? Mention it.

Reply:
It's added!